### PR TITLE
Json schema validation

### LINKED ISSUE #60: Provide option to validate responses against JSONSchema
Proposed syntax would be something along the lines of:
```
http.get('/something') {
    body.shouldBe validAgainstSchema('schema.json')
}
```

Things to consider:
* loading schema relative to current working dir
* config for location of specs

### PR DESCRIPTION
**Pull request by tsiq-karold**
Fixes #60

### CODE REVIEW

File: webtau-docs/webtau/REST/JSON-schema.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+The example schemas are:
+
+:include-json: examples/schemas/valid-schema.json {title: "Correct schema for the example above"}
+:include-json: examples/schemas/invalid-schema.json {title: "Incorrect schema for the example above"}
+
+The second example above will fail with the following error message:
+
+```
+invalid schema (examples/scenarios/rest/jsonSchema/validateSchema.groovy)
+> executing HTTP GET http://192.168.1.5:53830/weather
```
Comment (MykolaGolubyev) [inline webtau-docs/webtau/REST/JSON-schema.md]: for docs sake shall we put some human readable host here?

File: webtau-docs/webtau/REST/JSON-schema.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
+# Validation
+
+Webtau supports validation of objects against [JSON Schema](https://json-schema.org/).  For example, to validate the
+entire response body use the `complyWithSchema` matcher as follows:
+
+:include-groovy: scenarios/rest/jsonSchema/validateSchema.groovy {bodyOnly: true, title: "Example schema validation"}
```
Comment (MykolaGolubyev) [inline webtau-docs/webtau/REST/JSON-schema.md]: will it make sense to include an example where the not the entire body is validated?

File: webtau-json-schema/pom.xml [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+            <version>1.9.2</version>
+            <exclusions>
+                <exclusion>
+                    <groupId>joda-time</groupId>
+                    <artifactId>joda-time</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+
+        <dependency>
```
Comment (MykolaGolubyev) [inline webtau-json-schema/pom.xml]: what is it used for?

File: webtau-docs/webtau/REST/JSON-schema.md (hunk 1 above)
Comment (tsiq-karold) [reply webtau-docs/webtau/REST/JSON-schema.md]: Yes, it does, I just copied it straight from Maven output but I'll make it friendlier.

File: webtau-json-schema/pom.xml (hunk 3 above)
Comment (tsiq-karold) [reply webtau-json-schema/pom.xml]: I had to add this to make enforcer happy.  Joda is a transitive dep for both the json schema validation library and open api validation library.  They use subtly different versions, 2.9.4 vs. 2.9.7.  This forces them onto 2.9.7 and stops enforcer whinging.

I'll add a comment.

**Pull request merged**